.gitignore

```diff
@@ -2,6 +2,7 @@
 **/lib/**
 **/etc/**
 *.zip
+node_modules
```

Comment:
I still have the directories locally (and others might too). This prevents them from sneaking back into the repo (which happened to me). They don't hurt anything in the `.gitignore` anyhow.